add Radicle Integrations & Tooling Org (#300)
better-internet/radworks-docs: docs/community/01_ecosystem.md

roles that are better suited to a legal entity, such as the maintenance of Radworks' intellectual property and
initiation of Radworks' legal analysis and action. All work done by the Foundation Org for Radworks is funded by the treasury via an annual Org proposal. See the Foundation Org's [community approved strategy and budget for 2024](https://community.radworks.org/t/formal-review-rgp-21-foundation-org-proposal-2024/3420/3).

> The Better Internet Foundation does not control treasury funds, nor can it direct product development (Radicle & Drips).

## Grants Org

The [Radworks Grants](https://github.com/radicle-dev/radicle-grants) Org finds, funds, and guides research and
development of any FOSS projects that fall outside the purview of currently funded product/engineering orgs (e.g.
Radicle, Drips). See the Grants Org's [community approved strategy & budget for 2024](https://community.radworks.org/t/formal-review-rgp-20-grants-org-proposal-2024-v2/3464).

## Radicle Integrations & Tooling Org

> The focus of this Org is to integrate the Radicle Forge with other developer tools. They aim to lower friction for developers who want to migrate to Radicle, by allowing them to keep their existing workflows. These intergations enable smoother user onboarding (i.e. quicker adoption) as well as user retention. See the Radicle Integrations & Tooling Org's [community approved strategy & budget for 2024](https://community.radworks.org/t/formal-review-rgp-23-start-the-radicle-integrations-tooling-org-2024/3508). 

Reviewer: ```suggestion
The Radicle Integrations & Tooling Org focus is to integrate the Radicle Forge with other developer tools to lower friction for developers who want to migrate to Radicle, by allowing them to keep their existing workflows. These integrations support smoother user onboarding (i.e. quicker adoption) to Radicle as well as long-term user retention. See the Radicle Integrations & Tooling Org's [community approved strategy & budget for 2024](https://community.radworks.org/t/formal-review-rgp-23-start-the-radicle-integrations-tooling-org-2024/3508). 
```
